src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,52 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "cinemaHall")

```

Comment:
SQL databases use snake case commonly, not camel case  
```suggestion
@Table(name = "cinema_hall")
```